frontend/apps/reader/readerui.lua

```diff
@@ -342,6 +349,16 @@ function ReaderUI:init()
     end
     self.postInitCallback = nil
 
+    -- Now that document is loaded, store book metadata in settings
+    -- (so that filemanager can use it from sideCar file to display
+    -- Book information).
+    -- via pcall because picdocument:getProps() may fail
+    local ok, doc_props = pcall(self.document.getProps, self.document)
+    if not ok then
+        doc_props = {}
+    end
+    self.doc_settings:saveSetting("doc_props", doc_props)
```

Comment:
If you do not want to work on splitting the configuration file, I am OK with this. The configuration file is badly less optimized. We have to update it. https://github.com/koreader/koreader/issues/2789